tests/integration_tests/lossy_ddl/run.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -eu
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=cdc.test
+SINK_TYPE=$1
+
+MAX_RETRIES=10
+
+# Because we want the lossy DDL to not cause any data updates, so we can check the
+# data in the blackhole sink to see if any row is updated.
+function check_lossy_ddl() {
+	# Check finish_mark is written to the log.
+	is_finish_mark_exist=$(grep "BlackHoleSink: DDL Event" "$1/cdc.log" | grep -c "finish_mark")
```

Comment:
Should we check the total number of ddl events here?

Reply:
No, we didn't change its behavior.